docker/mkdocs/Dockerfile

```diff
@@ -1,7 +1,5 @@
-FROM            node:16
```

Comment:
Using a ready-to-use Docker image is way easier than trying to install MkDocs ourselves, especially on the new Ubuntu with more strict security measures for `apt-get`.